package.json

```diff
@@ -23,13 +23,13 @@
   "main": "index.js",
   "repository": "https://github.com/tidepool-org/seagull-client.git",
   "dependencies": {
-    "amoeba": "0.4.1",
-    "lodash": "3.4.0",
+    "amoeba": "0.6.0",
+    "lodash": "4.17.11",
```

Comment:
Looks like `lodash` can be removed altogether if you ditch https://github.com/tidepool-org/seagull-client/blob/master/lib/client.js#L33 as well.